test/e2e/testing-manifests/ingress/neg-exposed/svc.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: hostname
+  annotations:
+    cloud.google.com/neg: '{"ingress":true,"exposed_negs":{"80":{}}}'
```

Comment:
i would recommend you expose 2 ports. For instance, expose 80 and 443. And ingress reference 80.

That would result in 2 NEGs